src/main/java/org/helpberkeley/memberdata/ControlBlock.java

```diff
@@ -382,22 +382,32 @@ private void processAuditControl(String variable, String value, long lineNumber)
     }
 
     //
-    // An OpManager data field should look like "userName | phone number"
+    // An OpManager data field should look like "username" or "userName | phone number"
     //
     private void processOpsManager(final String value, long lineNumber) {
 
         String[] fields = value.split("\\" + INTRA_FIELD_SEPARATOR, -42);
 
-        if (fields.length != 2) {
-            throw new MemberDataException("OpsManager value \"" + value
-                    + "\" at line " + lineNumber + " does not match \"username | phone\".\n");
-        }
+//        if (fields.length != 2) {
+//            throw new MemberDataException("OpsManager value \"" + value
+//                    + "\" at line " + lineNumber + " does not match \"username | phone\".\n");
+//        }
 
         String userName = fields[0].trim();
-        String phone = fields[1].trim();
+        String phone = "";
 
         StringBuilder errors = new StringBuilder();
 
+        if (fields.length == 2) {
+            phone = fields[1].trim();
+            if (phone.isEmpty()) {
+                errors.append("OpsManager value \"" + value + "\" at line ").append(lineNumber).append(" does not match \"username\" or \"username | phone\"").append(".\n");
```

Comment:
My personal preference is not to let lines get too long. I don't want to have to horizontally scroll to see them, and auto-line wrapping display is ugly to me because it breaks the visual indentation of the code.  In the way-long-ago 80 columns was a standard width limit.  These days I try to keep Java lines less than 120 characters.  As this line is using a [fluent interface](https://en.wikipedia.org/wiki/Fluent_interface), instead of just breaking it in one place, I would break it at each of the .append calls.  Example:
```
    errors.append("xyzzy")
        .append("something")
        .append("something more")
        .append("something final");
```